lib/rubocop/cop/performance/sum.rb

```diff
@@ -98,7 +98,7 @@ def build_good_method(init)
 
           unless init.empty?
             init = init.first
-            good_method += "(#{init.source})" if init.source.to_i != 0
+            good_method += "(#{init.source})" unless init.int_type? && init.source.to_i.zero?
```

Comment:
`init.source.to_i.zero?` => `init.value.zero?`

Reply:
It's definitely simple. Thank you for your review!